apps/docs-app/docs/features/deployment/overview.md

```diff
@@ -51,37 +51,33 @@ export default defineConfig({
 
 ## Deploying with a Custom URL Prefix
 
-If you are deploying with a custom URL prefix, such as https://domain.com/`basehref`/ you must do these steps for [server-side-data-fetching](https://analogjs.org/docs/features/data-fetching/server-side-data-fetching), [html markup and assets](https://angular.io/api/common/APP_BASE_HREF), and [dynamic api routes](https://analogjs.org/docs/features/api/overview) to work correctly on the specified `basehref`.
+If you are deploying with a custom URL prefix, such as https://domain.com/ `basehref` you must do these steps for [server-side-data-fetching](https://analogjs.org/docs/features/data-fetching/server-side-data-fetching), [html markup and assets](https://angular.io/api/common/APP_BASE_HREF), and [dynamic api routes](https://analogjs.org/docs/features/api/overview) to work correctly on the specified `basehref`.
 
-1. Instruct Angular on how recognize and generate URLs. Create a new file `app.config.env.ts`.
+1. Update the `app.config.ts` file to use the new file.
+
+This instructs Angular on how recognize and generate URLs.
 
 ```ts
-import { ApplicationConfig } from '@angular/core';
```

Comment:
Why was this removed?

Reply:
I got rid of app.config.env.ts with mergeApplicationConfig in favor of using import.meta.env.BASE_URL

Reply:
Yes, but `ApplicationConfig` is still being used in the code snippet

Reply:
Got it! Done.